pkgs/by-name/fi/filebot/package.nix

```diff
@@ -45,6 +51,7 @@ stdenv.mkDerivation (finalAttrs: {
     curlWithGnuTls
     libmms
     glib
+    zenity
```

Comment:
The zenity package does not contain any libraries or headers, why is this here?

```suggestion
```

Reply:
I probably misunderstood where it should go. I'll be happy to look deeper into this and make the correct changes though.

Reply:
Looking back at this, I saw that libmediainfo is here which (I know it's a lib) is a depdendency of the program, so I assumed this is where an executable that needs to exist on the path should be as well. Perhaps the path is where, instead. I shall remove this.

Reply:
buildInputs are where dependencies go that are used during the build and linked against at runtime.

Executables that need to exist in `$PATH` *during the build* go into `nativeBuildInputs`.

Executables that need to exist in `$PATH` at runtime need to be manually shoved into the wrappers of the executables.

Reply:
This is very concise. Thank you. Let me rework this based on all the feedback ya'll have provided. I'll push changes today or tomorrow. Probably tomorrow as I'll need to learn about the rpath patching.